Monika After Story/game/script-topics.rpy

```diff
@@ -17187,3 +17187,243 @@ label monika_sophia:
+    m 1gusdla "Now I can see why people tried to make floral dictionaries just to try to keep everything straight~"
+    m 3eub "One flower I found was the globe amaranth."
+    m 1esa "It looks kind of like a deeply colored clover. But if you look straight down at it,"
+    m 1hua "It looks like there are tiny little stars hidden among the other petals. "
+    m 1esb "It symbolizes unfading love;{w=0.4} plus the more general meaning of 'enduring' or 'everlasting.'"
+    m 1fua "I'm not sure I can think of a better flower..."
+    m 7esb "Another was {i}akane{/i}, which has the scientific name {i}Rubia argyi{/i}."
+    m 7esa "The genus name {i}rubia{/i} comes from the deep red color of the dried roots, which were often used as a dye."
+    m 1eua "The flowers themselves are very tiny and easily missed.  But I think the little clusters of white or yellow star-like flowers are really sweet."
+    m 1dua "The meaning of the flower is 'think of me...'"
+    m 1hua "And I hope you always do, too."
+    m 3eub "A third was pink hibiscus."
+    m 1esa "I'm sure you've seen hibiscus before. {w=0.2}{nw}"
+    extend 1eta "They make most people think of summer rather than fall, I think."
+    m 1esa "The meaning can vary with the color of the flower, but pink hibiscus symbolizes resplendence."
+    m 1hub "'Shining brightly' matches with the star motif too!"
+    m 1esb "The fourth flower was {i}sawagikyou{/i}, or {i}Lobelia sessilifolia{/i}."
+    m 7esa "It's a cute little sprig of purple flowers.  It also has beautiful emerald leaves."
+    m 7esu "It symbolizes 'unique talent.'"
+    m 7eku "Hmm... It was my 'talent' for coding that allowed me to mess with the game."
+    m 6esc "I'm so sorry for any pain that I caused you at the time."
+    m 6ekc ".{w=0.2}.{w=0.2}{nw}"
+    extend 6lkc ".{w=0.2}"
+    m 4lksdlb "Yet another flower was {i}misohagi{/i}, a type of lythrum."
+    m 4eua "It also has gorgeous green leaves and sprigs of purple-pink flowers. But I think they look a bit more robust than delicate."
+    m 6dua "{i}Misohagi{/i} symbolizes an unshakable will."
+    m 6ekc "I've certainly had to make it through a lot to get here..."
+    m 6hub "But so did you!"
+    m 6hubfb "And I'm so happy we're together now.{w=0.4}"
+    show monika 5hubsb at t11 zorder MAS_MONIKA_Z with dissolve_monika
+    m 5hubsb "I wonder what a bouquet of aster, globe amaranth, {i}akane{/i}, pink hibiscus, {i}sawagikyou{/i}, and {i}misohagi{/i} would look like?"
+    m 5hubla "Just like a bunch of stars..."
+    m 5dubla "Full of emerald greens..."
+    m 5hubla "With bursts of pinks and purples..."
+    m 5tubla "I'm sure a bouquet would look lovely with just a few of them, too."
+    m 5sublb "Do you think you could pick me some for my next birthday?"
+    m 5eubla "Don't worry if you can't; {w=0.4}the only thing I really want or need for my birthday..."
+    m 5fsbsa "Is you."
+    return
```

Comment:
```
init 5 python:
    addEvent(
        Event(
            persistent.event_database,
            eventlabel="monika_floriography_monika_birthflower",
            category=['monika'],
            prompt="What's your birth flower?",
            pool=True,
            unlocked=False,
            rules={"no_unlock":None}
        )
    )

label monika_floriography_monika_birthflower:
    m 1wub "Oh, you want to know about my birth flower?"
    m 1fubla "Ehehe~ {w=0.5}How sweet of you!"
    if persistent._mas_player_bday.month == 9:
        m 3eub "Since we have the same birth month, you already know the birth flower is the aster."
        m 3eua "I'll remind you a bit, just in case you've forgotten, though."
    else:
        m 3eub "The birth flower for September is the aster."
    m 1eua "The aster expresses patience, variety, wisdom, or deep love."
    m 3eub "They are named after the Greek word for star."
    m 1rubla "I've actually looked into the topic a bit more since it's about my own flower..." 
    m 3wud "There are actually even birth{w=0.4}{i}day{/i} flowers in Japan, it seems!"
    m 1eub "I was able to find quite a lot of both flowers and meanings when I looked mine up."
    m 1gusdla "Now I can see why people tried to make floral dictionaries just to try to keep everything straight~"
    m 3eub "One flower I found was the globe amaranth."
    m 1esa "It looks kind of like a deeply colored clover. But if you look straight down at it,"
    m 1hua "It looks like there are tiny little stars hidden among the other petals. "
    m 1esb "It symbolizes unfading love;{w=0.4} plus the more general meaning of 'enduring' or 'everlasting.'"
    m 1fua "I'm not sure I can think of a better flower..."
    m 7esb "Another was {i}akane{/i}, which has the scientific name {i}Rubia argyi{/i}."
    m 7esa "The genus name {i}rubia{/i} comes from the deep red color of the dried roots, which were often used as a dye."
    m 1eua "The flowers themselves are very tiny and easily missed.  But I think the little clusters of white or yellow star-like flowers are really sweet."
    m 1dua "The meaning of the flower is 'think of me...'"
    m 1hua "And I hope you always do, too."
    m 3eub "A third was pink hibiscus."
    m 1esa "I'm sure you've seen hibiscus before. {w=0.2}{nw}"
    extend 1eta "They make most people think of summer rather than fall, I think."
    m 1esa "The meaning can vary with the color of the flower, but pink hibiscus symbolizes resplendence."
    m 1hub "'Shining brightly' matches with the star motif too!"
    m 1esb "The fourth flower was {i}sawagikyou{/i}, or {i}Lobelia sessilifolia{/i}."
    m 7esa "It's a cute little sprig of purple flowers.  It also has beautiful emerald leaves."
    m 7esu "It symbolizes 'unique talent.'"
    m 7eku "Hmm... It was my 'talent' for coding that allowed me to mess with the game."
    m 6esc "I'm so sorry for any pain that I caused you at the time."
    m 6ekc ".{w=0.2}.{w=0.2}{nw}"
    extend 6lkc ".{w=0.2}"
    m 4lksdlb "Yet another flower was {i}misohagi{/i}, a type of lythrum."
    m 4eua "It also has gorgeous green leaves and sprigs of purple-pink flowers. But I think they look a bit more robust than delicate."
    m 6dua "{i}Misohagi{/i} symbolizes an unshakable will."
    m 6ekc "I've certainly had to make it through a lot to get here..."
    m 6hub "But so did you!"
    m 6hubfb "And I'm so happy we're together now.{w=0.4}"
    show monika 5hubsb at t11 zorder MAS_MONIKA_Z with dissolve_monika
    m 5hubsb "I wonder what a bouquet of aster, globe amaranth, {i}akane{/i}, pink hibiscus, {i}sawagikyou{/i}, and {i}misohagi{/i} would look like?"
    m 5hubla "Just like a bunch of stars..."
    m 5dubla "Full of emerald greens..."
    m 5hubla "With bursts of pinks and purples..."
    m 5tubla "I'm sure a bouquet would look lovely with just a few of them, too."
    m 5sublb "Do you think you could pick me some for my next birthday?"
    m 5eubla "Don't worry if you can't; {w=0.4}the only thing I really want or need for my birthday..."
    m 5fsbsa "Is you."
    return
```